src/LaunchDarkly.Client/LaunchDarkly.Client.nuspec

```diff
@@ -14,5 +14,8 @@
     <releaseNotes>Switch to asynchronous HTTP client</releaseNotes>
     <copyright>$copyright$</copyright>
     <tags>Feature Configuration</tags>
+     <dependencies>
+      <dependency id="Newtonsoft.Json" version="8.0.3" />
```

Comment:
Before this change: Importing the LD client into your app would bring in an unknown version of NewtonSoft- maybe the wrong one if you already had an older one in a local repo.

After this change: We are specifying the specific version to use.